src/cloud-code/Parse.Cloud.js

```diff
@@ -0,0 +1,111 @@
+var ParseCloudExpress = require("parse-cloud-express");
+ParseCloudExpress.Parse.Cloud.httpRequest = require("../httpRequest");
```

Comment:
I just merged your change to httprequest on parse-cloud-express and published 1.2.0.